src/components/Layout/Navbar.js

```diff
@@ -1,10 +1,12 @@
-import {Navbar} from "react-bootstrap";
-
+import { useContext } from "react";
+import { Navbar } from "react-bootstrap";
+import AppContext from "../../store/AppContext";
 const Nav = () => {
+    const { user } = useContext(AppContext);
     return (
         <nav>
             <Navbar bg={'light'}>
-                <Navbar.Brand to={'/'}>Affirmations</Navbar.Brand>
+                <Navbar.Brand to={'/'}>{user && user.name? `HI ${user.name}!`:"Affirmations"}</Navbar.Brand>
```

Comment:
What is the point of checking for user and user.name? If a user is not there wouldn't it show Affirmations anyways?